modulus/models/domino/model.py

```diff
@@ -257,53 +257,57 @@ class GeometryRep(nn.Module):
     def __init__(self, input_features, model_parameters=None):
         super().__init__()
         geometry_rep = model_parameters.geometry_rep
+        radii = geometry_rep.geo_conv.radii
+        # num_scales = len(radii)
+
+        self.bq_warp = nn.ModuleList()
+        self.geo_processors = nn.ModuleList()
+        for j, p in enumerate(radii):
+            self.bq_warp.append(
+                BQWarp(
+                    input_features=input_features,
+                    grid_resolution=model_parameters.interp_res,
+                    radius=radii[j],
+                )
+            )
+            self.geo_processors.append(
+                GeoProcessor(
+                    input_filters=geometry_rep.geo_conv.base_neurons_out,
+                    model_parameters=geometry_rep.geo_processor,
+                )
+            )
 
-        self.bq_warp_short = BQWarp(
-            input_features=input_features,
-            grid_resolution=model_parameters.interp_res,
-            radius=geometry_rep.geo_conv.radius_short,
-        )
-
-        self.bq_warp_long = BQWarp(
-            input_features=input_features,
-            grid_resolution=model_parameters.interp_res,
-            radius=geometry_rep.geo_conv.radius_long,
-        )
-
-        self.geo_conv_out = GeoConvOut(
-            input_features=input_features,
-            model_parameters=geometry_rep.geo_conv,
-            grid_resolution=model_parameters.interp_res,
-        )
-
-        self.geo_processor_short_range = GeoProcessor(
-            input_filters=geometry_rep.geo_conv.base_neurons_out,
-            model_parameters=geometry_rep.geo_processor,
-        )
-        self.geo_processor_long_range = GeoProcessor(
-            input_filters=geometry_rep.geo_conv.base_neurons_out,
-            model_parameters=geometry_rep.geo_processor,
-        )
+        self.geo_conv_out = nn.ModuleList()
+        for j, p in enumerate(radii):
```

Comment:
Same here - `j` and `p` are not used here at all. Can do something like:
```py
for _ in range(len(radii)):
```
or just fold this loop into the previous one.